README.md

```diff
@@ -34,15 +34,15 @@ const {create} = require ('sanctuary');
 const {env, FutureType} = require ('fluture-sanctuary-types');
 const {resolve} = require ('fluture');
 
-const S = create ({checkTypes: true, env: S.env.concat (env)});
+const S = create ({checkTypes: true, env: $.env.concat (env)});
```

Comment:
Let's do this:

```suggestion
const S = create ({checkTypes: true, env: env.concat (env)});
```

And then add `{env,}` to the imports from `"sanctuary"`.

Reply:
We cannot name two things `env`. ;)

Reply:
Yeah, I noticed that too. Perhaps we can just stop the destructuring, like done in the esm example.